yarn.lock

```diff
@@ -13115,7 +13115,7 @@ graphql@15.7.2:
   resolved "https://registry.yarnpkg.com/graphql/-/graphql-15.7.2.tgz#85ab0eeb83722977151b3feb4d631b5f2ab287ef"
```

Comment:
Looking at this ☝️ , it seems the old graphql version is still a dependency of something, so I assume I made an error and did not upgrade whatever package causes this dependency